### DISCUSSION TITLE
Is Umami compliant with the GPDR?

### DISCUSSION THREAD

**Discussion by MagicLike** [umami-software/umami#890] [Q&A] [answered]
I want to integrate Umami into my website and since I live in the EU, I wanted to know if Umami is compliant with the GDPR.
Also, do I need to write a privacy policy, if so, what needs to be included?

**Comment by jallbrit** [comment]
https://umami.is/docs/faq

**Reply by MagicLike** [reply]
Unfortunately I already read that after I opened this discussion...
But still thank you!
However do you also know if I have to write any information in a privacy policy or at least a note that I use Umami?

**Reply by discoverlance-com** [reply]
This is also a question I have, and I nearly opened up a discussion about it till I saw this. The FAQ does well to mention that there's no cookie stored but a bit worried about privacy policy though if it needs to be included in such.

**Reply by MagicLike** [reply]
I have since solved it by simply writing down that no personal data is collected, but among other things [List data collected here] is collected.

**Comment by ptman** [comment]
How does Umami identify users so that every request doesn't appear as a new user? Not using cookies?

**Comment by viharm** [comment, marked as answer]
There is a slightly wider question about what to include in the privacy policy of websites using _Umami_, irrespective of whether they use the cloud or the self-hosted version.

There is no doubt that some information is collected and stored to generate the analytics. The GDPR compliance comes from the fact that none of it is personal information of the visitors.

However there needs to be some guidance on what we should put in our privacy policies on our websites.

Here's what I have put on my site which uses a self-hosted _Umami_ instance; as a starter. But since I'm not an expert, I'd like some opinions and feedback:

> ### Privacy
> This site does not collect any personal data
> 
> - This site uses a first-party, privacy-friendly system for basics analytics to understand how this site is used
> - This site collects only the basic information which is presented by the browser or device you are using to access this website and other websites
> - None of the information collected can be used to personally identify any person or device access this site
> - The information collect is not shared with any third party for any reason

---

DISCLAIMER: I am not qualified to recommend what should be used for your uses. Each application will have a slightly different policy based on the application, deployment, visitors, etc.

Please only use the above as starting point, and please customize to your liking. Depending on the risk assessment, I strongly recommend seeking professional legal guidance to ensure that the policy you use, covers the use case.

**Reply by seedy** [reply]
Thanks for your guidance about legal mentions 🙏

**Comment by viharm** [comment]
DISCLAIMER: I am not qualified to recommend what should be used for your uses. Each application will have a slightly different policy based on the application, deployment, visitors, etc.

Please only use the above as starting point, and please customize to your liking. Depending on the risk assessment, I strongly recommend seeking professional legal guidance to ensure that the policy you use, covers the use case.

**Comment by MagicLike** [comment]
please take a look at #2929
